### DISCUSSION TITLE
AWS CLI support?

### DISCUSSION THREAD

**Discussion by vdrover** [isontheline/pro.webssh.net#1130] [Ideas]
Hi there, 

I recently made some changes to my AWS infrastructure and now I must use the AWS CLI to SSH into my instance. Once the CLI is installed, you run the `aws configure` command to store your "keys" on your system.

The SSH command can then connect to your instance. 

I don't think WebSSH is built to support AWS CLI. For example, the cli does not use a hostname which is required for an SSH profile in WebSSH.:

`ssh username@instanceID -o ProxyCommand='aws ec2-instance-connect open-tunnel --instance-id instanceID --region=eu-west-1'`

Any thoughts on supporting AWS CLI?

**Comment by isontheline** [comment]
Hi Victor 👋

No plan yet to support AWS cli as WebSSH uses plain SSH connections instead.

ProxyCommand with aws ec2 seems to use a HTTPS Proxy in order to establish the SSH connection.
I'm searching how to use a such AWS HTTPS Proxy without the need of the aws cli

**Comment by vdrover** [comment]
Thanks for the update. I'm following with interest!
